OpenRA.Mods.Common/MapGenerator/MultiBrush.cs

```diff
@@ -117,19 +121,19 @@ public MultiBrushInfo(MiniYaml my)
 							throw new YamlException($"Invalid MultiBrush Weight `{node.Value.Value}`");
 						break;
 					case "Actor":
-						actors.Add(new ActorInfo(node.Value));
+						actorsAcc.Add(new ActorInfo(node.Value));
 						break;
 					case "BackingTile":
-						if (TerrainTile.TryParse(node.Value.Value, out var backingTile))
-							BackingTile = backingTile;
+						if (TerrainTile.TryParse(node.Value.Value, out var backingTile2))
```

Comment:
```suggestion
						if (TerrainTile.TryParse(node.Value.Value, out var bt))
```

maybe just call it `bt`. Using a number is confusing

Reply:
Done